modules/virt-example-vm-node-placement-node-affinity.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc
+
+[id="virt-example-vm-node-placement-node-affinity_{context}"]
+= Example: VM node placement with node affinity
+
+In this example, the VM must be scheduled on a node that has the label `example.io/example-key = example-value-1` or the label `example.io/example-key = example-value-2`. The constraint is met if only one of the labels is present on the node. If neither label is present, the VM is not scheduled.
+
+If possible, the scheduler avoids nodes that have the label `example-node-label-key = example-node-label-value`. However, if all nodes have this label, the scheduler ignores this constraint.
```

Comment:
I wonder... 

However, if all candidate nodes have this label, the scheduler ignores this constraint.

----

Just trying to avoid confusion. Maybe I'm being too litera and reading too much into "all nodes"l. point being the only nodes that matter are the ones that already met the required nodeLabel criteria.